From e4ef13166118fe991f7590b093671f9361e9c1cc Mon Sep 17 00:00:00 2001
From: fujiwarat <takao.fujiwara1@gmail.com>
Date: Tue, 16 Mar 2021 22:13:01 +0900
Subject: [PATCH] data: Change default input mode to Hiragana (rhbz#1938129)

GNOME desktop sets unnecessary JP keymap by default and a few newcomers
wish to use Hiragana mode directly.
This internal patch will be deleted if the default desktop would be changed.
---
 data/org.freedesktop.ibus.engine.anthy.gschema.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.freedesktop.ibus.engine.anthy.gschema.xml.in b/data/org.freedesktop.ibus.engine.anthy.gschema.xml.in
index 773be09..cd5a13b 100644
--- a/data/org.freedesktop.ibus.engine.anthy.gschema.xml.in
+++ b/data/org.freedesktop.ibus.engine.anthy.gschema.xml.in
@@ -3,7 +3,7 @@
   <schema path="/org/freedesktop/ibus/engine/anthy/common/"
           id="org.freedesktop.ibus.engine.anthy.common">
     <key name="input-mode" type="i">
-      <default>3</default>
+      <default>0</default>
       <summary>Input Mode</summary>
       <description></description>
     </key>
-- 
2.28.0

